src/globals.js

```diff
@@ -31,9 +31,9 @@ export const TracerGlobals = (() => {
     Object.assign(handlerInputs, { event: {}, context: {} });
 
   const setTracerInputs = ({
-    token = '',
+    token = process.env.LUMIGO_TRACER_TOKEN || '',
     debug = false,
```

Comment:
Nice, change the debug mechanism to work also in this way.
This is a better way but its weird that we have 2 ways to handle env vars inputs